tests/torchtune/utils/test_precision.py

```diff
@@ -58,12 +58,18 @@ def test_error_bf16_unsupported(self, mock_verify):
 
     @pytest.mark.skipif(not cuda_available, reason="The test requires GPUs to run.")
     def test_set_float32_precision(self) -> None:
+        setattr(  # noqa: B010
+            torch.backends, "__allow_nonbracketed_mutation_flag", True
```

Comment:
Is this setting change contained within the test? If not, I we should grab the original value and set it back to that at the end of the test.